internal/clearingway/handlers.go

```diff
@@ -118,10 +118,11 @@ var MenuCommand = &discordgo.ApplicationCommand{
 	DefaultMemberPermissions: &adminPermission,
 	Options: []*discordgo.ApplicationCommandOption{
 		{
-			Type:        discordgo.ApplicationCommandOptionString,
-			Name:        "menu",
-			Description: "Menu to send",
-			Required:    true,
+			Type:         discordgo.ApplicationCommandOptionString,
+			Name:         "menu",
+			Description:  "Menu to send",
+			Required:     true,
+			Autocomplete: true,
```

Comment:
will there be any situation where we would want AutoComplete to be false? idk if we want it to be configurable?

Reply:
oh this just means this option has an autocomplete feature implemented (like the autocomplete for the world field in `/clears`), makes it easier to know what menus are send-able